doc/source/tutorial/interpolate/1D.rst

```diff
@@ -182,6 +182,61 @@ linear extrapolation since ``make_interp_spline`` extrapolates by default. Consi
 See :ref:`the extrapolation section <tutorial-extrapolation>` for more details
 and discussion.
 
+.. _tutorial-interpolate_batching:
+
+Batches of `y`
+==============
+
+Univariate interpolators accept not only one-dimensional ``y`` arrays, but also
+``y.ndim > 1``. The interpretation is that ``y`` is a *batch* of 1D data arrays: by
+default, the zeroth dimension of `y`` is the interpolation axis, and the trailing 
+dimensions are batch dimensions.
+One way of thinking about it is to imagine a collection (a *batch*) of functions,
+:math:`f(x), g(x), h(x), \dots`, sampled at the points :math:`x_j`, such that the ``y``
+array represents a collection of :math:`f(x_j), g(x_j), h(x_j),\dots`, stacked along
+``axis=1``.
```

Comment:
After seeing this rendered, I realized that while it can be infered, it may not be immediately apparent whether we are stacking the functions or the points. To be explicit, maybe:
```suggestion
Consider a collection (a *batch*) of functions :math:`f_j` sampled at the points
:math:`x_i`. We can instatiate a single interpolator for all of these functions by 
providing a two-dimensional array ``y`` such that ``y[i, j]`` records :math:`f_j(x_i)`.
```

If desired, could add something like:
> In other words, each row contains the values of all functions a single point, and each column contains the values of a single function at all points.